Ty

Relten98 opened the pull request:
Updated Readme, Reworked other elements. Currently working on the main tavern page, and getting that up and running so it can be in by tonight.

Relten98 committed:
Update App.js

Relten98 committed:
more fixes

Relten98 committed:
Fixed my broken avatar frame

Relten98 committed:
Merge branch 'main' into Ty

Relten98 committed:
Fixed broken stuff, added new images.

Relten98 committed:
Merge branch 'main' into Ty

Relten98 committed:
Fixed main Tavern screen, now PROPERLY mobile responsive.

Relten98 committed:
Basic pages, start of the finalized, generated sheet.

Relten98 committed:
Merge branch 'main' into Ty

Relten98 committed:
Updated

Updated Readme,

Created initial component for the generated sheet,

Created the initial component for the X button.

Relten98 committed:
New components, and an update to set up the main page.

Relten98 committed:
Why is there a head merge in Ranger.js?

charlestbell reviewed (changes requested):
Says there are conflicts with main. Do another pull and resolve whatever there is

Relten98 committed:
Merge branch 'main' into Ty

Relten98 committed:
Merge branch 'main' into Ty

Relten98 committed:
Update index.js

Relten98 committed:
f

charlestbell reviewed (changes requested):
There are conflicts with main. 

I checked out this branch and pulled down and ran it. A few things I found:

- This branch is missing the updated Feats tab.
- This branch is missing several things I updated on the main tavern page.

Could you checkout main and pull down and run it and see what it looks like? Thanks bro!

Relten98 committed:
Merge branch 'main' into Ty

Relten98 committed:
Committed

Relten98 committed:
Merge branch 'main' into Ty

Relten98 committed:
Create Style.css

Relten98 committed:
Update App.js

Relten98 committed:
CSS Styling

Relten98 committed:
Update style.css

List items now are now more obvious on what the user is hovering over, along with making it easier for users to click the text in order to gain the information

charlestbell committed:
Made it not break when database returns null
